docs/source/status.rst

```diff
@@ -44,6 +44,8 @@ Data Types
 +-------------------+-------+-------+-------+------------+-------+-------+-------+
 | Decimal128        | ✓     | ✓     |       |            |       |       | ✓     |
 +-------------------+-------+-------+-------+------------+-------+-------+-------+
+| Decimal256        | ✓     | ✓     |       |            |       |       |       |
```

Comment:
@quinnj Is Decimal256 supported on Julia?

Reply:
Not yet, but I started the work locally, just need to find time to finish it up. It should be pretty easy to at least allow deserializing/serializing, but more work to make the Decimal256 useful for arithmetic, etc.